Sources/NodesSSO/NodesSSOController.swift

```diff
@@ -1,5 +1,6 @@
 import Crypto
 import Foundation
+import Sugar
```

Comment:
was this by accident?

Reply:
and if so, is Sugar used anywhere else?

Reply:
Sugar is used in this file for the `redirectURL` stuff